sda-commons-server-morphia/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
   compile project(':sda-commons-server-dropwizard')
+  compile project(':sda-commons-shared-certificates')
```

Comment:
You seem to use an outdated version of sda-commons. The master branch now uses Gradle 7. Instead of using `compile` scope here, you should use `api`. It's probably a good idea to rebase your branches.